src/App.tsx

```diff
@@ -1,61 +1,113 @@
+import React, { useState } from 'react';
 import './App.scss';
+import { Todo, FullTodo } from './react-app-env';
+import { TodoList } from './components/TodoList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+const fullTodos: Todo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: usersFromServer.find(user => user.id === todo.userId),
+}));
+
+export const App: React.FC = () => {
+  const [todos, setTodos] = useState(fullTodos);
+  const [user, setUser] = useState(0);
```

Comment:
```suggestion
  const [userId, setUserId] = useState(0);
```
Better store user id, not index